### PR TITLE
Update README for newbies

### PR DESCRIPTION
**Pull request by Motiejus**
* Prepare ringpop for development.
* Update the max open file limit (needed in default settings of OSX).
* Introduce tick-cluster to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,12 +5,34 @@ and provides request forwarding as a routing convenience. It can be used to
...
+ulimit -n 4096
+```
+
+# Tick Cluster
+An example application `scripts/tick-cluster.js` is included to this
+repository. It just launches a ringpop cluster of a given size. Using this
+application is the quickest way to start a ringpop cluster.
+
+```
+./scripts/tick-cluster.js --help
```
Comment (thanodnl) [inline README.md]: Can you add the command to start `tick-cluster` with the application in `main.js` it might not be as straight forward as one hopes.

Comment (Motiejus) [inline README.md]: Like discussed, I changed help line of `tick-cluster` to make it easier to understand what the interpreter is instead.

Comment (dansimau) [inline README.md]: +1 to Nils' original comment here. Don't make the reader think.

Comment (Motiejus) [inline README.md]: OK, let it be so. I uploaded a new revision.

**Comment by dansimau** [comment]
LGTM :+1:

**Comment by thanodnl** [comment]
lgtm

File: README.md [hunk 2]
```diff
@@ -5,12 +5,34 @@ and provides request forwarding as a routing convenience. It can be used to
...
 
 # Installation
-To install Ringpop:
+To install Ringpop for usage as a library:
 
 ```
 npm install ringpop
 ```
 
+Prepare the current directory for development:
```
Comment (jwolski) [inline README.md]: I think this is a bit confusing. We should make clear distinction between Installation as a User and Setup for a Developer. The way the `npm install ringpop` and `npm install` instructions flow makes it seem like both are required.

I typically see these broken into 2 sections:

```
# Installation
```

and 

```
# Developing
```

**Pull request merged**